spec/classes/foreman_proxy__tftp__netboot_spec.rb

```diff
@@ -41,13 +41,21 @@
         case facts[:operatingsystem]
         when /^(RedHat|Scientific|OracleLinux)$/
           it { should contain_file("/tftproot/grub2/grubx64.efi").with_source('/boot/efi/EFI/redhat/grubx64.efi') }
-          it { should contain_file("/tftproot/grub2/shim.efi").with_source('/boot/efi/EFI/redhat/shim.efi').with_owner('root').with_mode('0644') }
+          if facts[:os]['release']['major'].to_i >= 8
+            it { should contain_file("/tftproot/grub2/shimx64.efi").with_source('/boot/efi/EFI/redhat/shimx64.efi').with_owner('root').with_mode('0644') }
```

Comment:
is this related to GH-592?

Reply:
Yes, good catch. The other PR is to essentially drop shim.efi and use only shimx64.efi (both are available on EL7). If that is merged, it would simplify the tests here because there would no longer be a difference between EL7 and EL8.

I think we could go ahead and merge this, since the other one is waiting on contrib with a few requested changes, and the other one would then also need to recombine the EL{7,8} tests to simply test for shimx64.efi in all cases. What do you think @mmoll ?

Reply:
I'd prefer to have other PR in first, tbh...